_data/mentors.yml

```diff
@@ -86,8 +92,12 @@
 - name: Anshuman Verma
   image: ansh.jpg
   twitter: Anshumaniac12
+ 
+- name: Rahul Tyagi
```

Comment:
My name is Rahul Tiwari and twitter handle is invinciblycool
Please make the changes